usecases/objects/delete.go

```diff
@@ -21,8 +21,15 @@ import (
 )
 
 // DeleteObject Class Instance from the conncected DB
-func (m *Manager) DeleteObject(ctx context.Context, principal *models.Principal, id strfmt.UUID) error {
-	err := m.authorizer.Authorize(principal, "delete", fmt.Sprintf("objects/%s", id.String()))
+//
+// if class == "" it will delete all object with same id regardless of the class name.
+// This is due to backward compatibility reasons and should be removed in the future
```

Comment:
love the helpful comments

Reply:
Thanks @parkerduckworth! I think it's always good practice to document why we take a decision in order to make life easier for other developers and ourselves included